includes/admin/views/html-admin-settings.php

```diff
@@ -83,7 +83,13 @@
 				 * @since 1.8
 				 */
 				do_action( self::$setting_filter_prefix . '_tabs' );
-				?>
+
+				// Show link to New Reports page
+				$isReports = isset($_GET['page']) && $_GET['page'] === 'give-reports' ? true : false;
```

Comment:
@henryholtgeerts I think we can use on `&&` operator because it returns `true` or `false`, no need of trinary condition:

```
$isReports = isset($_GET['page']) && $_GET['page'] === 'give-reports' 
```

Reply:
See: https://github.com/impress-org/givewp/pull/4338/commits/eedcdb06d97378828caf999e15c064a49cc537ad